INSTALLATION.md

```diff
@@ -0,0 +1,203 @@
+# Map GL Style Build - Installation Guide
+
+:wave: This guide walks through installing and setting up `map-gl-style-build` in a new or existing project containing one or more [Mapbox GL](https://docs.mapbox.com/mapbox-gl-js/style-spec/) or [Maplibre GL](https://maplibre.org/maplibre-style-spec) stylesheet.
+
+## Prerequisites
+
+### Required Software
+- **Node.js** (v16 or higher recommended)
```

Comment:
I've been adding `.nvmrc` to most of our carto-tools projects keep us on the same page in terms of node versions, and I wonder if we should do the same for this repo? (I see you mention `nvm` further down in this README 👍)